test/service/test_request_validation.py

```diff
@@ -487,3 +487,10 @@ def test_bdbag_manifest(self):
                     'fastq[read2].dos_url',
                     'fastq[read2].file_url',
                 ], reader.fieldnames)
+
+    def test_manifest_format_validation(self):
+        for manifest_endpoint in '/manifest/files', '/repository/files/export':
+            with self.subTest(manifest_endpoint=manifest_endpoint):
+                url = self.base_url + f'{manifest_endpoint}?format=invalid-type'
+                response = requests.get(url)
+                self.assertEqual(400, response.status_code, response.content)
```

Comment:
Why do you use the `response.content` as the assertion error message? Please explain.

Reply:
While coding this test, this assert kept getting 500 response errors. Adding `response.content` as the assertion error message helped me diagnose the source of those errors.